src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -3799,7 +3799,10 @@ export class StudyViewPageStore {
                     }
                     const attributeChanged =
                         isDefaultAttr &&
-                        !_.isEqual(toJS(attribute), initDataBinFilter);
+                        !_.isEqual(
```

Comment:
we can probably just merge this, but would it not be better to test for what we actually want to compare?

Reply:
as in, what properties of the attribute need to be different in order for this to be regarded as changed

Reply:
we need to compare for everything except `showNA`